backend/infrahub/config.py

```diff
@@ -281,7 +281,6 @@ class DatabaseSettings(BaseSettings):
     query_size_limit: int = Field(
         default=5_000,
         ge=1,
-        le=20_000,
```

Comment:
We don't want to set some other new upper limit just to have something? 100_000?

Reply:
I don't think it makes sense, this is not a value users should change randomly in production.
And sometimes we want to test very high values for a short period of time, having no limit set would allow such tests.